client/specs_support/helper.js

```diff
@@ -27,7 +26,7 @@ export default class Helper{
     }, settings);
 
     return configureStore({
-      settings: Immutable.fromJS(initialSettings)
+      settings: initialSettings
```

Comment:
Incidentally, the value of `settings` didn't seem to affect the tests at all.  Perhaps this method could be removed?